pkgs/top-level/perl-packages.nix

```diff
@@ -210,6 +210,31 @@ let
      };
   };
 
+  AlienSDL = buildPerlModule {
+    pname = "Alien-SDL";
+    version = "1.446";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/F/FR/FROGGS/Alien-SDL-1.446.tar.gz";
+      sha256 = "c9aa2c9dc3c63d89773c7d7203f2a46d1b924d0c72d9f801af147a3dc8bc512a";
+    };
+    patches = [ ../development/perl-modules/alien-sdl.patch ];
+
+    installPhase = ''
+      runHook preInstall
```

Comment:
`runHook preInstall` `runHook postInstall` are not needed.
I see that it is a copy-paste from `buildPerlModule`s `installPhase`, the reason for runHooks there are for one can supply `preInstall` and `postInstall` as `buildPerlModule`'s argument.